src/Microsoft.Extensions.Caching.Redis/RedisCache.cs

```diff
@@ -78,10 +78,16 @@ public void Set(string key, byte[] value, DistributedCacheEntryOptions options)
 
             var creationTime = DateTimeOffset.UtcNow;
 
+            var absoluteExpiration = options.AbsoluteExpiration;
+            if (options.AbsoluteExpirationRelativeToNow.HasValue)
+            {
+                absoluteExpiration = creationTime + options.AbsoluteExpirationRelativeToNow;
```

Comment:
Should this absoluteExpiration be passed into GetExpirationInSeconds to reduce redundant logic there?